deploy/scripts/Create-IomtWebJobsEnvironment.ps1

```diff
@@ -0,0 +1,109 @@
+<#
+.SYNOPSIS
+Creates a new IoMT FHIR Connector for Azure without using Stream Analytics
+.DESCRIPTION
+#>
+param
+(
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [ValidateLength(5,12)]
+	[ValidateScript({
+        if ("$_" -cmatch "(^([a-z]|\d)+$)") {
+            return $true
+        }
+        else {
+			throw "Environment name must be lowercase and numbers"
+            return $false
+        }
+    })]
+    [string]$EnvironmentName,
+
+    [Parameter(Mandatory = $false)]
+    [ValidateSet('Australia East','East US','East US 2', 'West US', 'West US 2','North Central US','South Central US','Southeast Asia','North Europe','West Europe','UK West','UK South')]
+    [string]$EnvironmentLocation = "North Central US",
+
+    [Parameter(Mandatory = $false)]
+    [ValidateSet('R4')]
+    [string]$FhirVersion = "R4",
```

Comment:
Since we only support R4, do we need this parameter?

Reply:
It is not really needed at this time but the other scripts like the sandbox one have it. I think it was added there for some of the investigation work to support R5